nxontology_data/efo/queries/sources.rq

```diff
@@ -0,0 +1,17 @@
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+
+SELECT ?efo_id ?xref  ?axiom_source
+WHERE {
```

Comment:
I wonder if there are cases of xrefs that don't have any axioms, i.e. things matched by

https://github.com/related-sciences/nxontology-data/blob/c7b14299b18ed391762758bf76d8f5f806006a60/nxontology_data/efo/queries/xrefs.rq#L28-L29

We could do this match first and then make the axiom match OPTIONAL. Or we could decide this query is only for getting xref sources and we don't care about anything without a source.

Questions:

1. Do all `oboInOwl:hasDbXref` triples have corresponding axioms?
2. Do all axioms with `owl:annotatedProperty oboInOwl:hasDbXref` have corresponding `oboInOwl:hasDbXref` triples?

Reply:
> I wonder if there are cases of xrefs that don't have any axioms, i.e. things matched by

There are cases like that, for example `MONDO:0004947` in `EFO:0000094` and `ICD10:O35` in `EFO:0009682` don't have axioms.



> We could do this match first and then make the axiom match OPTIONAL. Or we could decide this query is only for getting xref sources and we don't care about anything without a source.

I think that this query should be only for getting xref sources